src/App.jsx

```diff
@@ -138,6 +140,14 @@ export default function App() {
               </RequireAuth>
             }
           />
+          <Route
+            path="/urlblacklist"
```

Comment:
cho dễ đọc cái url path thì mình dùng kebab case nha em 
```suggestion
            path="/url-blacklist"
```